pandas/core/arrays/base.py

```diff
@@ -560,16 +571,13 @@ def _concat_same_type(cls, to_concat):
         """
         raise AbstractMethodError(cls)
 
-    @property
-    def _can_hold_na(self):
-        # type: () -> bool
-        """Whether your array can hold missing values. True by default.
+    _can_hold_na = True
+    """Whether your array can hold missing values. True by default.
```

Comment:
Not sure what our recommended style is here. You can probably just change it to a comment.